src/chem_parse/ChemlabParser.py

```diff
@@ -229,7 +231,8 @@ def p_form_manage(self, p):
 
     def p_unit(self, p):
         '''Unit : UnitTok
-                        | PrefixTok Unit'''
+                        | PrefixTok
```

Comment:
Im okay with just passing the prefix but we really need to test this well.

Reply:
Also you changed the rule but did not update the logic. I think you meant to only pass in either a unit, a prefix, or a prefix followed by a unit, not the unit rule but the unit token